Resources/Prototypes/_CM14/Access/Groups/medical_access_groups.yml

```diff
@@ -0,0 +1,21 @@
+﻿- type: accessGroup
+  id: CMCECMO
```

Comment:
you forget to remove "CE"
CM **CE** CMO

Reply:
thanks